plugins/node/opentelemetry-instrumentation-pg/test/pg-pool.test.ts

```diff
@@ -288,5 +290,139 @@ describe('pg-pool@2.x', () => {
         assert.strictEqual(resNoPromise, undefined, 'No promise is returned');
       });
     });
+
+    describe('when specifying a responseHook configuration', () => {
+      const dataAttributeName = 'pg_data';
+      const query = 'SELECT 0::text';
+      const events: TimedEvent[] = [];
+
+      describe('AND valid responseHook', () => {
+        beforeEach(async () => {
+          instrumentation.disable();
+          instrumentation = new PgInstrumentation({
+            enhancedDatabaseReporting: true,
```

Comment:
Please add `setConfig` to the instrumentation class to avoid re-creating it every time